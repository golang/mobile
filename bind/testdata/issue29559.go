package issue29559

type AString = string

func TakesAString(s AString) {}
