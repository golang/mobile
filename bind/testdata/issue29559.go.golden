// Package main is an autogenerated binder stub for package issue29559.
//   gobind -lang=go issue29559
//
// File is generated by gobind. Do not edit.
package main

/*
#include <stdlib.h>
#include <stdint.h>
#include "seq.h"
#include "issue29559.h"

*/
import "C"

import (
	_seq "golang.org/x/mobile/bind/seq"
	"issue29559"
)

// suppress the error if seq ends up unused
var _ = _seq.FromRefNum

//export proxyissue29559__TakesAString
func proxyissue29559__TakesAString(param_s C.nstring) {
	_param_s := decodeString(param_s)
	issue29559.TakesAString(_param_s)
}
